packages/neutrino-preset-react/package.json

```diff
@@ -21,9 +21,10 @@
     "eslint-plugin-react": "^6.10.3",
     "neutrino-middleware-loader-merge": "^5.8.0",
     "neutrino-preset-web": "^5.8.0",
-    "react-hot-loader": "^3.0.0-beta.6"
+    "react-hot-loader": "^3.0.0-beta.6",
+    "deepmerge": "^1.3.2"
```

Comment:
We don't need deepmerge as we already have `neutrino-middleware-loader-merge` here.